Fixed typos, images, added missed setting on customer engagement page (#1675)
magento/merchdocs: src/reports/customer-engagement.md

   * 48 hours
   * 72 hours

   ![System Alerts]({% link reports/assets/system.alerts.block.png %}){: .zoom}
   _System Alerts_

> ## PWA Settings
> 
> Specify **PWA Storefront Base URL** field if you use a PWA storefront URL which is different from Magento's default frontend base URL. e.g. _https://pwa.yourdomain.com/_.
> 
>    ![System Alerts]({% link configuration/dotdigital/assets/developer-dash-pwa-settings.png %}){: .zoom}
>    _PWA Base URL_

Reviewer: ```suggestion
## PWA settings

If you use a PWA storefront URL, specify the **PWA Storefront Base URL** (for example, `https://pwa.yourdomain.com/`). This is different from the default store frontend base URL.

![System Alerts]({% link configuration/dotdigital/assets/developer-dash-pwa-settings.png %}){: .zoom}
_PWA storefront base URL_
```

Author: Hello @leslietilling , thank you for the review and noticing. I have commited suggested changes and also deleted the previously added image.
Thanks!
